### PR TITLE
Influxdb monitoring

### LINKED ISSUE #8: Create monitoring notifications from influx(without webhook for now)
Create function for getting info, about some metric or not:

> lb_timing, 
> lb_down_test
> iscsi_connection
> ce_result

return error if no metrics for last 5 minutes

influx1.eco.tsi-dev.otc-service.com
Db: csm
User: csm

### PR DESCRIPTION
**Pull request by lego963**
Resolves #8

### CODE REVIEW

File: delatore/sources/influx.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                             CSM_RDS_TEST=get_time_delta(client, 'ce_result'))
+        return proof_of_work
+
+
+def get_time_delta(client, entity):
+    query = f"SELECT LAST(*) FROM {entity} LIMIT 1;"
+    try:
+        last_record = client.query(query)
+        last_time = last_record.raw['series'][0]['values'][0][0]
+        time_parts = last_time.split('.');
```
Comment (outcatcher) [inline delatore/sources/influx.py]: `;` is not required

File: delatore/sources/influx.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        time_zone = ''
+        for item in time_parts[1]:
+            if item.isalpha():
+                time_zone += item
+        second_part = '.123456' + time_zone
+        last_time_ms = datetime.datetime.strptime(time_parts[0] + second_part, f'%Y-%m-%dT%H:%M:%S.%f{time_zone}')
+        if (datetime.datetime.utcnow() - last_time_ms).total_seconds() < 300:
+            return "OK"
```
Comment (outcatcher) [inline delatore/sources/influx.py]: Double quotes mixed with single

File: delatore/json2mdwn.py [hunk 3]
```diff
@@ -2,6 +2,12 @@
 import re
 
 
+def _normalize_key(src: str):
+    if src.isupper():
```
Comment (outcatcher) [inline delatore/json2mdwn.py]: Maybe skip only `.title()` for upper case keys? Still replacing `_` with ` ` for them

File: delatore/emoji.py [hunk 4]
```diff
@@ -7,6 +7,7 @@ class Emoji(NamedTuple):
     RUNNING = '🏃'
     SUCCESS = '✅'
     CANCELED = '⛔'
+    NO_DATA = '⚠'
```
Comment (outcatcher) [inline delatore/emoji.py]: Let's use ❔ for NO_DATA

File: delatore/json2mdwn.py (hunk 3 above)
Comment (lego963) [reply delatore/json2mdwn.py]: No

**Pull request merged**